### PR TITLE
Fix: Persist snippet tabs

### LINKED ISSUE #1948: Snippet tab does not persist between the page and the expanded modal
**Describe the bug**
Code snippet tab does not persist after clicking the Expand button. The snippet also disappears after closing the modal.

**To Reproduce**
1. Go to https://developer.microsoft.com/en-us/graph/graph-explorer
2. Click on Code snippets
3. Click on the JavaScript tab
4. Click on Expand
5. Notice that the tab opened on the expanded modal is different

To reproduce snippet disappearing after closing the modal:
1. Click on the Java tab
2. Now click expand to open the modal
3. Close the modal
4. Notice that the response is 'Snippet not found'

**Expected behavior**
The opened tab should be consistent between the page and the expanded modal

**Screenshots**
image

After opening modal
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Onokaev**
## Overview
Fixes #1948 

### Demo
Normal snippet tab
image

Modal snippet tab
image

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Click on the Code snippets tab and select Powershell
Click on Expand tab
Notice that the snippet tab on the opened modal is still Powershell
Now close the Modal
Notice that the snippet is still displayed

### CODE REVIEW

**Comment by millicentachieng** [comment]
I noticed double and sometimes triple vertical scroll bars on the right of the expanded modal. all of which are unnecessary. The horizontal scroll bar is also unnecessary. This PR  didn't introduce the bug, so I've created this [issue](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/1981) for tracking.

File: src/app/views/query-response/pivot-items/pivot-items.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { queryResponseStyles } from '../queryResponse.styles';
 import { Response } from '../response';
 import { Snippets } from '../snippets';
 
-export const getPivotItems = () => {
+export const GetPivotItems = () => {
```
Comment (millicentachieng) [inline src/app/views/query-response/pivot-items/pivot-items.tsx:22]: Why the change of function name from camel case?

Comment (Onokaev) [reply src/app/views/query-response/pivot-items/pivot-items.tsx:22]: The function is returning components and the recommended convention is to have all of these functions camel-cased. SonarCloud also flagged it as a bug
image

https://sonarcloud.io/project/issues?resolutions=FIXED&pullRequest=1958&id=microsoftgraph_microsoft-graph-explorer-v4&open=AYJYe_3T5cxs88NX26G2

**Review by ElinorW — APPROVED** [review]

File: src/app/views/query-response/pivot-items/pivot-items.tsx (hunk 1 above)
Comment (millicentachieng) [reply src/app/views/query-response/pivot-items/pivot-items.tsx:22]: You mean pascal-cased?

**Review by millicentachieng — APPROVED** [review]

**Pull request merged**